### PR TITLE
Comments on sample

### PR DESCRIPTION
**Pull request by nantunes**

### CODE REVIEW

File: docs/platform/visual/index.md [hunk 1]
```diff
@@ -11,10 +11,9 @@ The Pentaho Visualization API provides a unified way to visualize data across th
 [Analyzer](http://www.pentaho.com/product/business-visualization-analytics), 
 [PDI](http://www.pentaho.com/product/data-integration), 
 [CDF](http://community.pentaho.com/ctools/cdf/)).
-Essentially, it is a set of abstractions that ensures total isolation between 
-applications, 
-visualizations and 
-configurations (the glue).
+
+Essentially, it is a set of abstractions that ensures isolation between
+applications, visualizations and configurations.
```
Comment (nantunes) [inline docs/platform/visual/index.md]: I think the glue metaphor isn't that clear. We understand it because that's what we've been calling it. Maybe "that glue the two together" or just remove it.

File: docs/platform/visual/index.md [hunk 2]
```diff
@@ -34,21 +33,17 @@ Visualizations are implemented on top of the Pentaho Core, Type and Data JavaScr
   as well as with other client-side component frameworks.
 - Using [Core APIs]()
   provides visualizations with features such as localization, theming and registration.
-  
-The use of [Pentaho Platform JavaScript APIs]({{site.refDocsUrlPattern | replace: '$', 'pentaho'}}) for anything not 
-directly related with the visualization concept, 
-ensures developers can reuse knowledge that they gain in other components of the Pentaho suite 
-to build visualizations and vice-versa.
```
Comment (nantunes) [inline docs/platform/visual/index.md:41]: I bit confusing (I didn't understood it). Can you rephrase it? Is it needed?

File: docs/platform/visual/index.md [hunk 3]
```diff
@@ -34,21 +33,17 @@ Visualizations are implemented on top of the Pentaho Core, Type and Data JavaScr
...
 
-The Pentaho Visualization API comes with a set of stock visualizations, 
-covering the most common chart-types.
-Most stock visualizations are based on the [CCC](http://community.pentaho.com/ctools/ccc/) charting library,
-which means that the stock visualizations are _super-configurable_ to suite your organization's preferred style.
+The platform provides a set of stock visualizations, covering the most
+common chart-types.
+Based on the [CCC](http://community.pentaho.com/ctools/ccc/) charting
+library, they're customizable and extensible to fit your organization's
+desired look and feel.
```
Comment (nantunes) [inline docs/platform/visual/index.md:41]: **Super** configurable seems a bit self praising. Maybe a bit less marketing? And should we link to configuration documentation, specially if we talk about where to put the CCC extension points?

**Comment by nantunes** [comment]
@dcleao Some are commented through GitHub, other are comments inserted in the code (so, **do not merge this PR**, of course).

File: docs/platform/visual/index.md (hunk 2 above)
Comment (dcleao) [reply docs/platform/visual/index.md:41]: This recalls the central position of the Platform JavaScript APIs. It says to developers that the knowledge they gain in these can be capitalized in multiple components of the Pentaho suite. When you learn these APIs while developing vizs, you'll gain knowledge that you'll be able to use when developing CDF dashboards, and vice-versa. When you learn these APIs while developing vizs, you'll gain knowledge that you'll be able to use when developing vizs.

It also frames the strategical positioning of the Visualization API, of leaving out any abstractions not strictly related with visualization.

I think this is relevant. Maybe someone can provide a clearer formulation.

File: docs/platform/visual/index.md (hunk 1 above)
Comment (carlosrusso) [reply docs/platform/visual/index.md]: Do we need the sentence at all?

**Review by dcleao — APPROVED** [review]
Added commit with changes.

File: docs/platform/visual/index.md (hunk 1 above)
Comment (dcleao) [reply docs/platform/visual/index.md]: Do you think that the one sentence that summarizes what the visualization API is is not needed?

Comment (carlosrusso) [reply docs/platform/visual/index.md]: Yes, I do not think that sentence is absolutely necessary in this file. The previous sentence states the purpose, which is enough for framing the user's mind.

That sentence can be safely moved to the article that provides more details about the viz api.

Comment (dcleao) [reply docs/platform/visual/index.md]: I disagree quite a lot.

Comment (carlosrusso) [reply docs/platform/visual/index.md]: I don't have such strong feelings about this particular sentence. I still doubt the strict need for having it, but it does not shock me to have it there. I don't think the matter is worthy of further  discussion.
